packages/pigeon/platform_tests/README.md

```diff
@@ -0,0 +1,25 @@
+# Native Pigeon Tests
+
+This directory contains native test harnesses for native and end-to-end tests
+of Pigeon-generated code. The [test script](../tool/run_tests.dart) generates
+native code from [pigeons/](../pigeons/) into the native test scaffolding, and
+then drives the tests there.
+
```

Comment:
nit: Might be worth explicitly saying that's how test here are executed.

Reply:
Done.